wallet/build.gradle

```diff
@@ -64,35 +70,27 @@ dependencies {
     implementation project(path: ':common')
     implementation 'com.google.android.gms:play-services-auth:18.1.0'
 
-    def room_version = "2.2.5"
-    implementation "androidx.room:room-ktx:$room_version"
-    implementation "androidx.room:room-runtime:$room_version"
-    annotationProcessor "androidx.room:room-compiler:$room_version"
+    // Database
+    implementation "androidx.room:room-ktx:$roomVersion"
+    implementation "androidx.room:room-runtime:$roomVersion"
+    kapt "androidx.room:room-compiler:$roomVersion"
 
-    //Uphold Integration
+    // Uphold Integration
     implementation project(path: ':uphold-integration')
 
     implementation 'androidx.multidex:multidex:2.0.1'
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "androidx.core:core-ktx:1.3.2"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.8"
 
-    implementation 'com.github.bumptech.glide:glide:4.11.0'
-    annotationProcessor 'com.github.bumptech.glide:compiler:4.11.0'
-
-    implementation 'com.github.bumptech.glide:glide:4.11.0'
-    annotationProcessor 'com.github.bumptech.glide:compiler:4.11.0'
+    implementation "com.github.bumptech.glide:glide:$glideVersion"
+    kapt "com.github.bumptech.glide:compiler:$glideVersion"
 
     implementation 'com.github.MikeOrtiz:TouchImageView:3.0.3'
 
-    // Import the BoM for the Firebase platform
-    implementation platform('com.google.firebase:firebase-bom:26.1.1')
-
     // Declare the dependencies for the Crashlytics and Analytics libraries
     // When using the BoM, you don't specify versions in Firebase library dependencies
-    implementation 'com.google.firebase:firebase-crashlytics'
-    implementation 'com.google.firebase:firebase-analytics'
+    implementation platform("com.google.firebase:firebase-bom:$firebaseVersion")
```

Comment:
Are crashlytics and analytics included elsewhere?

Reply:
Yes, crashlytics and analytics go to the common module since AnalyticsService is located there, given that we're gonna use events and possibly crash logging in liquid and uphold modules too. BoM is still left here for inappmessaging and dynamic links, although it might be the case that it is only required in the common module now.